### DISCUSSION TITLE
Where is lib/index.js?

### DISCUSSION THREAD

**Discussion by dejan7** [tailwindlabs/tailwindcss#9823] [General]
Hey there, i'm examining the structure of the package to learn more about stuff and in the `package.json` i see

```
  "bin": {
    "tailwind": "lib/cli.js",
    "tailwindcss": "lib/cli.js"
  },
```

However i do not see where are these actual files in the repo itself?! Maybe i'm missing something obvious, but any clues are helpful. Thanks

**Comment by wongjn** [comment]
These files are built via the `prepublishOnly` `script` defined in the `package.json`:
https://github.com/tailwindlabs/tailwindcss/blob/f2f1ee9b5585d11945587f688a4c533c5287aaf7/package.json#L20

**Reply by adamwathan** [reply]
Yep, here's the compiled file that is published to npm:

https://unpkg.com/browse/tailwindcss@3.2.4/lib/index.js

**Reply by dejan7** [reply]
Ah clear, i was just browsing through git repo. Thanks!
